backend/fms_core/tests/test_template_importers/add_sample_metadata_test.py

```diff
@@ -0,0 +1,86 @@
+from django.test import TestCase
+from django.contrib.contenttypes.models import ContentType
+import datetime
+
+from fms_core.template_importer.importers import SampleAddMetadataImporter
+from fms_core.tests.test_template_importers._utils import load_template, APP_DATA_ROOT
+
+from fms_core.models import Sample, SampleKind, SampleMetadata
+
+from fms_core.services.container import create_container
+from fms_core.services.sample import create_full_sample
+
+
+class SampleMetadataTestCase(TestCase):
+    def setUp(self) -> None:
+        self.importer = SampleAddMetadataImporter()
+        self.file = APP_DATA_ROOT / "Sample_metadata_v3_8_0.xlsx"
+        ContentType.objects.clear_cache()
+
+        self.sample1_name = 'SampleTestMetadata'
+        self.sample2_name = 'Sample2TestMetadata'
+        self.sample_metadata1_name = "MetadataName1"
```

Comment:
If you add the lower case conversion, test also that the name is lower(name). You can also test the case of the space replacement by underscore.